.rhdh/bundle/manifests/rhdh-operator.csv.yaml

```diff
@@ -42,7 +42,7 @@ metadata:
     features.operators.openshift.io/token-auth-gcp: "false"
     repository: https://gitlab.cee.redhat.com/rhidp/rhdh/
     support: Red Hat
-    skipRange: '>=1.0.0 <1.2.5'
+    skipRange: '>=1.2.4 <1.3.1'
```

Comment:
the 1.2.5 operator should not try to skip the 1.3.1 operator. This is wrong.

Reply:
if anything the problem could be this line https://github.com/redhat-developer/rhdh-operator/blob/release-1.3/.rhdh/bundle/manifests/rhdh-operator.csv.yaml#L360 which makes 1.3 a replacement for 1.1, not 1.2.

Reply:
see this change https://github.com/redhat-developer/rhdh-operator/commit/90e90cf76db80961fec5c156af7247a0a44ad821